extra-wm/i3-gaps/autobuild/defines

```diff
@@ -0,0 +1,9 @@
+PKGNAME=i3-gaps
+PKGSEC=x11
+PKGDEP="dmenu libev libxkbcommon pango perl-anyevent-i3 perl-json-xs \
+        startup-notification xcb-util-cursor xcb-util-keysyms \
+        xcb-util-wm yajl xcb-util-xrm"
+PKGRECOM="i3lock i3status i3blocks"
+BUILDDEP="graphviz doxygen xmlto"
+PKGCONFL="i3"
```

Comment:
I tend to add `PKGCONFL=` in a different section with the rest, but that's purely my habit. It's fine.

However, please also add `PKGCONFL="i3-gaps"` in `extra-wm/i3`.